sale_product_variants/models/sale_order.py

```diff
@@ -98,6 +98,8 @@ def onchange_product_template(self):
                 self.product_template.product_variant_ids[0])
         else:
             self.product_id = False
+            self.product_uom = self.product_template.uom_id
```

Comment:
Me refiero la otra parte del if (`self.product_id = (self.product_template.product_variant_ids and self.product_template.product_variant_ids[0])`) si desencadena el poner el resto de datos. Si no, habría que lanzar el onchange_product_id ahí.